client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -184,11 +213,50 @@ class DomainItem extends PureComponent {
 		return withoutHttp( site.URL );
 	}
 
+	renderSiteMeta() {
+		const { domainDetails, isManagingAllSites, site, translate } = this.props;
+
+		if ( isManagingAllSites ) {
+			return (
+				<div className="domain-item__meta">
+					{ translate( 'Site: %(siteName)s', {
+						args: {
+							siteName: this.getSiteName( site ),
+						},
+						comment:
+							'%(siteName)s is the site name and URL or just the URL used to identify a site',
+					} ) }
+				</div>
+			);
+		}
+
+		if ( ! isManagingAllSites && domainDetails.isWPCOMDomain ) {
```

Comment:
This conditional `! isManagingAllSites` seems redundant since we would have exited the function if `isManagingAllSites` was true.

Reply:
Yeah, thanks for catching that :)